wip-0001.md

```diff
@@ -28,18 +28,18 @@ The WIP process begins with a new idea for Witnet. Each potential WIP must have
 Small enhancements or patches to a particular piece of software often don't require standardisation between multiple projects; these don't need a WIP and should be injected into the relevant project-specific development workflow with a patch submission to the applicable issue tracker.
 Additionally, many ideas have been brought forward for changing Witnet that have been rejected for various reasons.
 The first step should be to search past discussions to see if an idea has been considered before, and if so, what issues arose in its progression.
-After investigating past work, the best way to proceed is by posting about the new idea to the [Witnet community Gitter chat room](https://gitter.im/witnet/community).
+After investigating past work, the best way to proceed is by posting about the new idea to the [Witnet community Discord server](https://gitter.im/witnet/community).
 
 Vetting an idea publicly before going as far as writing a WIP is meant to save both the potential author and the wider community time.
 Asking the Witnet community first if an idea is original helps prevent too much time being spent on something that is guaranteed to be rejected based on prior discussions (searching the internet does not always do the trick).
 It also helps to make sure the idea is applicable to the entire community and not just the author. Just because an idea sounds good to the author does not mean it will work for most people in most areas where Witnet is used.
 
-Once the champion has asked the Witnet community as to whether an idea has any chance of acceptance, a draft WIP should be presented to the [Witnet community Gitter chat room](https://gitter.im/witnet/community).
+Once the champion has asked the Witnet community as to whether an idea has any chance of acceptance, a draft WIP should be presented to the [Witnet community Gitter chat room][discord].
```

Comment:
Text needs changing here to "Witnet community Discord server"

Reply:
Fixed!